### PR TITLE
[WIP] [FEATURE] MS-72: Psychologist - Routes Refactor

### PR DESCRIPTION
**Pull request by Sabrucita**

### CODE REVIEW

**Review by JamirMansilla — APPROVED** [review]
Approved! 🌵

File: models/Candidates.js [hunk 1]
```diff
@@ -174,7 +174,7 @@ const CandidatesSchema = new Schema({
     },
   ],
   profileTypes: {
-    type: Schema.Types.ObjectId,
+    type: String,
```
Comment (BaltazarCortina) [inline models/Candidates.js:177]: Why did you change this? I think it was correct for it to be an ObjectId

**Review by EugenioPonzone — APPROVED** [review]

File: models/Candidates.js (hunk 1 above)
Comment (Sabrucita) [reply models/Candidates.js]: Yes, my mistake! I already fixed it 😊
